### PR TITLE
Changes for running LLDB test suite for Swift on PowerPC64LE

### PR DESCRIPTION
**Pull request by sarveshtamba**
The attached changes are required for running LLDB test suite related test cases for Apple Swift 5 on PowerPC64LE.

### CODE REVIEW

**Comment by sarveshtamba** [comment]
Hi @lilyball @penryu @nico @ismail can you please review this?

**Comment by sarveshtamba** [comment]
> Hi @lilyball @penryu @nico @ismail can you please review this?

Also, please let me know the correct branch to submit these PR changes in case this branch is not the correct one. Thanks in advance.

**Comment by nico** [comment]
Wrong nico?

**Comment by jasonmolenda** [comment]
I don't normally review this area of code, but these changes look fine.  Merging to stable is fine, it will automerge to branch upstream-with-swift.

**Comment by lilyball** [comment]
Wrong lily? Perhaps you meant @millenomi?

**Comment by sarveshtamba** [comment]
@dcci can you please review and approve this if it looks OK?

**Comment by dcci** [comment]
@swift-ci test

File: source/Target/SwiftLanguageRuntime.cpp
Comment (dcci) [inline source/Target/SwiftLanguageRuntime.cpp:2681]: This bit looks fine.

**Comment by dcci** [comment]
The change in `SwiftLanguageRuntime.cpp` looks fine. One between Saleem and Stefan should take a look at the changes in CMake before this goes in.

**Comment by dcci** [comment]
I know Stefan is OOO so @compnerd is the best person to review this.

File: tools/repl/swift/CMakeLists.txt
Comment (compnerd) [inline tools/repl/swift/CMakeLists.txt:13]: This is terrible, but, yes, this is the best we can do right now :-(.

**Review by compnerd — APPROVED** [review]

**Comment by compnerd** [comment]
@swift-ci please test Linux platform

File: tools/repl/swift/CMakeLists.txt
Comment (weliveindetail) [reply tools/repl/swift/CMakeLists.txt:13]: Just as a note: RPATHs can be set via properties like in https://github.com/apple/swift-lldb/blob/stable/cmake/modules/AddLLDB.cmake#L192 (but yes, I am OOO and that's probably not critical)

**Comment by compnerd** [comment]
@swift-ci please Linux platform

**Comment by compnerd** [comment]
@swift-ci please test Linux platform

**Comment by sarveshtamba** [comment]
@compnerd @swift-ci any updates on this?

**Comment by compnerd** [comment]
@swift-ci please test Linux platform

**Comment by dcci** [comment]
@sarveshtamba sorry for the delay. This should be unblocked now.

**Comment by dcci** [comment]
@swift-ci test and merge

**Pull request merged**
